tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -214,3 +225,24 @@ func wait4AllK8sNodesToBeUp(
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	}
 }
+
+// deletePodsInParallel deletes pods in a given namespace in parallel
+func deletePodsInParallel(client clientset.Interface, namespace string, pods []*v1.Pod, wg *sync.WaitGroup) {
```

Comment:
did you check if the deletion of pods was still happening when site failover happened?

Reply:
Yeah I checked, you can refer to this log: https://gist.github.com/Aishwarya-Hebbar/05654821df11262348233081f233d106